README.md

```diff
@@ -152,6 +152,18 @@ works like dial, but calls back with a [Connection State Machine](#connection-st
 Connection state machines emit a number of events that can be used to determine the current state of the connection
 and to received the underlying connection that can be used to transfer data.
 
+### `switch.dialer.connect(peer, options, callback)`
+
+a low priority dial to the provided peer. Calls to `dial` and `dialFSM` will take priority. This should be used when
+an application only wishes to establish connections to new peers, such as during peer discovery when there is a low
+peer count.
+
+- `peer`: can be an instance of [PeerInfo][], [PeerId][] or [multiaddr][]
+- `options`: Optional
+- `options.priority`: Number of the priority of the dial, defaults to 1.
```

Comment:
Should we start with higher numbered priorities to leave some wiggle room in case there are other use cases in future? eg HIGH=10, LOW=20, and then in future maybe there's a MEDIUM=15

Reply:
I can't think of anything now, but that doesn't mean there wont be :). I've added those as constants to the constants file, and am now referencing them for the options.

Reply:
oh ups, I did not see this review discussion and left a comment in this regard too